src/main/resources/db/migration/V6.2.0.2.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE termsofuseandaccess SET license_id = (SELECT license.id FROM license WHERE license.name = 'CC0 1.0'), termsofuse = 'CC0 Waiver'
```

Comment:
termsofuse should be null after the update

Reply:
I thought it would be nice to be able to find the ones changed after the fact. There are many rows with termsofuse not null with license_id set. I can change it if you want.

Reply:
Hmm - in the UI if you have something in terms of use and select a license, the terms of use are nulled. I don't think there should be a way to have a license and terms of use or related fields- they terms of use aren't displayed when a license is in use. (Terms of Access and fields related to that are fine). I'd suggest putting null in with this PR and perhaps opening an issue to address any other cases that exist (legacy and if there's a way to create new datasets with a license and terms of use we should close that hole.).

Reply:
changed to null